Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,258 @@
+            foreach (var targetImageFileName in TargetImageFileNames)
+            {
+                // Read target image files.
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Detect faces from image stream.
+                    IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream);
+                    if (detectedFaces == null || detectedFaces.Count == 0)
+                    {
+                        Console.WriteLine($"[Error] No face detected from image `{targetImageFileName}`.");
+                        return;
+                    }
+
+                    Console.WriteLine($"{detectedFaces.Count} faces detected from image `{targetImageFileName}`.");
+                    if (detectedFaces[0].FaceId == null)
+                    {
+                        Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for recognition purpose.");
+                        return;
+                    }
+
+                    // Add detected faceId to targetFaceIds.
+                    targetFaceIds.Add(detectedFaces[0].FaceId);
+                }
+            }
+
+            // Read source image file.
+            using (FileStream stream = new FileStream(Path.Combine("Images", SourceImageFileName), FileMode.Open))
+            {
+                // Detect faces from image stream.
+                IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream);
+                if (detectedFaces == null || detectedFaces.Count == 0)
+                {
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName}`.");
+                    return;
+                }
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName}`.");
+                if (detectedFaces[0].FaceId == null)
+                {
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for recognition purpose.");
+                    return;
+                }
+
+                // Find similar example of faceId to faceIds.
+                IList<SimilarFace> similarResults = await Client.Face.FindSimilarAsync(detectedFaces[0].FaceId.Value, null, null, targetFaceIds);
+                if (similarResults.Count == 0)
+                {
+                    Console.WriteLine($"No similar faces to {SourceImageFileName}.{Environment.NewLine}");
+                }
+
+                foreach (var similarResult in similarResults)
+                {
+                    Console.WriteLine($"Faces from {detectedFaces[0].FaceId} & {similarResult.FaceId} are similar with confidence: {similarResult.Confidence}.");
+                }
+            }
+        }
```

Comment:
Add `Console.WriteLine();` for better readability.
Same for other methods and other samples.

Reply:
done